src/components/Input/index.tsx

```diff
@@ -0,0 +1,60 @@
+import styled from '@emotion/styled';
+import { DefaultProps } from '@utils/types/DefaultProps';
+import { ChangeEventHandler, RefObject } from 'react';
+
+interface InputProps extends DefaultProps<HTMLInputElement> {
+  placeholder?: string;
+  ref?: RefObject<HTMLInputElement>;
+  onChange?: ChangeEventHandler<HTMLInputElement>;
+  isValid?: boolean;
+  id: string;
+  name?: string;
+}
+
+const Input = ({
+  placeholder = '입력하세요',
+  ref,
+  onChange,
+  isValid = true,
+  id,
+  name,
+  ...props
+}: InputProps) => {
+  return (
+    <InputContainer
+      placeholder={placeholder}
+      ref={ref}
+      onChange={onChange}
+      id={id}
+      name={name ? name : id}
+      {...{ isValid }}
+      {...props}
+    ></InputContainer>
+  );
+};
+
+const InputContainer = styled.input<Pick<InputProps, 'isValid'>>(
+  ({ theme, isValid }) => {
+    const { color: themeColor } = theme;
+    const { primary100, error, gray200 } = themeColor;
+    return {
+      width: '8rem',
+      height: '1.6rem',
+      border: isValid ? `0.07rem solid ${gray200}` : `0.1rem solid ${error}`,
+      borderRadius: '0.2rem',
+      outline: 'none',
+      '::placeholder': {
+        color: gray200,
+        fontSize: '0.8rem',
+      },
+      ':focus': {
+        border: isValid
+          ? `0.1rem solid ${primary100}`
+          : `0.1rem solid ${error}`,
```

Comment:
```suggestion
        border: `0.1rem solid ${isValid ? primary100 : error}`
```
앞에 rem solid는 중복되는데 삼항연산자를 여기에 사용하는건 가독성이 별로인가요?

Reply:
저두 이거 추천 !!!!

Reply:
아항 바꿀게여!